src/routes/Routes.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react'
+import { Route, useHistory } from 'react-router-dom'
+import Question from '../views/Question'
+import Home from '../views/Home'
+import Login from '../views/Login'
+import Signup from '../views/Signup'
+import AskQuestion from '../views/AskQuestion'
+import About from '../views/About'
+import NotFound from '../views/404'
+import Switch from 'react-bootstrap/esm/Switch'
+import ReactGA from 'react-ga'
+
+export default () => {
+  const history = useHistory()
+  ReactGA.initialize('UA-204918741-1')
+  useEffect(() => {
+    const host = window.location.hostname
+    // To not track activities in GA during development testing
+    if (host !== 'localhost') {
```

Comment:
I think you can safely do the check directly instead of saving a variable:
`if (window.location.hostname !== 'localhost') {`

That's a little clearer and avoids a var declaration.

Reply:
make sense 👍

Reply:
Make sense 👍